## request
float32 x
float32 y
float32 z
float32 alp
float32 t0
float32 tf
---
## response
float32 cur_x
float32 cur_y
float32 cur_z
float32 cur_alp
int64 result
